### ISSUE TITLE
Add CI config

### ISSUE THREAD

**Issue by PavelBal** [Open-Systems-Pharmacology/OSPSuite.ParameterIdentification#2]
@msevestre I moved the fork to OSP, you can set up the CI now.

**Comment by msevestre** [comment]
@PavelBal  I need verison 0.9 from rCLR. Where can I find it?

**Comment by msevestre** [comment]
found it here
https://github.com/Open-Systems-Pharmacology/OSPSuite.ParameterIdentification/releases

**Comment by msevestre** [comment]
@PavelBal 

  there is no package called 'openxlsx'... what is this?

```
* installing *source* package 'ospsuite.parameteridentification' ...
** using staged installation
** R
** byte-compile and prepare package for lazy loading
Error in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]) : 
  there is no package called 'openxlsx'
Calls: <Anonymous> ... loadNamespace -> withRestarts -> withOneRestart -> doWithOneRestart
Execution halted
ERROR: lazy loading failed for package 'ospsuite.parameteridentification'
* removing 'C:/projects/ospsuite-parameteridentification/ospsuite.parameteridentification.Rcheck/ospsuite.parameteridentification'
```

**Comment by msevestre** [comment]
HELP!!
![image](https://user-images.githubusercontent.com/1041237/120228388-f9bf4e80-c218-11eb-85a5-95455ce74cf5.png)

What version of R should I install to load anything. So frustrating

**Comment by msevestre** [comment]
So...it looks like esqlab package depends on R 4.1?!? WHY
also why are the following dependencies required?

Imports: rClr, tools, **openxlsx**, parallel, **shiny, shinyjs**, hash

Can they be removed?

**Comment by msevestre** [comment]
Fixed with #7
